src/rag_processor/auth/dependencies.py

```
@@ -0,0 +1,58 @@
+"""FastAPI dependencies for authentication.
+
+Provides injectable dependencies for accessing the current user
+in protected endpoints.
+"""
+
+from __future__ import annotations
+
+from fastapi import HTTPException, Request, status
+
+from rag_processor.auth.models import CloudflareUser  # noqa: TC001 - Used at runtime
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `TC001`)

Remove unused `noqa` directive

(RUF100)